modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -80,7 +84,8 @@ AbaqusUMATStress::AbaqusUMATStress(const InputParameters & parameters)
     _number_external_properties(_external_property_names.size()),
     _external_properties(_number_external_properties),
     _external_properties_old(_number_external_properties),
-    _use_one_based_indexing(getParam<bool>("use_one_based_indexing"))
+    _use_one_based_indexing(getParam<bool>("use_one_based_indexing")),
+    _decomposition_method(getParam<MooseEnum>("decomposition_method"))
```

Comment:
```suggestion
    _decomposition_method(getParam<MooseEnum>("decomposition_method").getEnum<ComputeFiniteStrain::DecompMethod>())
```

and so on...

Reply:
Had to promote the enum to public in `ComputeFiniteStrain` to do this. Hopefully that's the right thing to do.